component/src/main/java/com/synopsys/integration/alert/component/diagnostic/model/NotificationTypeCount.java

```diff
@@ -0,0 +1,23 @@
+package com.synopsys.integration.alert.component.diagnostic.model;
+
+import com.synopsys.integration.alert.api.common.model.AlertSerializableModel;
+
+public class NotificationTypeCount extends AlertSerializableModel {
+
+    private static final long serialVersionUID = -7022752228360734890L;
+    private final String notificationType;
```

Comment:
Should NotificationTypeCount use the NotificationType enum rather than a String?

Reply:
To add to this, it appears the only place we use this is in the wait task where we use the NotificationType::name to compare.

Reply:
made the change